modules/tinymce/src/plugins/accordion/main/ts/core/Actions.ts

```diff
@@ -13,18 +13,21 @@ const insertAccordion = (editor: Editor): void => {
   const uid = Id.generate('acc');
   const summaryText = editor.dom.encode(editor.selection.getRng().toString() || editor.translate('Accordion summary...'));
   const bodyText = editor.dom.encode(editor.translate('Accordion body...'));
-  editor.insertContent(`<details data-mce-id="${uid}" class="mce-accordion" open="open"><summary class="mce-accordion-summary">${summaryText}</summary><p>${bodyText}</p></details>`);
 
-  const details = editor.dom.select(`[data-mce-id="${uid}"]`)[0];
-  if (!details) {
-    return;
-  }
-  details.removeAttribute('data-mce-id');
+  editor.undoManager.transact(() => {
```

Comment:
`insertContent` just calls the `mceInsertContent` command and all commands are in undo transactions, look topic in [Slack](https://cksource.slack.com/archives/C04KXE6PMRB/p1683319674177149?thread_ts=1683309133.416439&cid=C04KXE6PMRB
)